etc/grafana/dashboards/overview.json

```diff
@@ -4248,9 +4400,8 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "PBFA97CFB590B2093"
```

Comment:
you need to export the dashboard for sharing externally (see checkbox) 
![image](https://github.com/paradigmxyz/reth/assets/5773434/81e36b9a-b02f-4e0e-9fa3-ae1b0c6f075b)

Reply:
already fixed those conflicts earlier today, then jsvisa came with nicer fix, forgot to do it again 🙈